notebooks/official/feature_store/sdk-feature-store.ipynb

```diff
@@ -262,7 +262,15 @@
       },
```

Comment:
<p>replace with SDK call</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/790/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>Most of the featurestore services are in beta or older versions (v1, v1beta1). Can we use them?</p>

Reply:
I would recommend the SDK versions from this notebook:
https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/ml_ops/stage2/get_started_vertex_feature_store.ipynb